docs/api/classes/errors.functionconfigerror.html

```diff
@@ -112,7 +112,7 @@ <h3>constructor</h3>
 							<aside class="tsd-sources">
 								<p>Overrides Error.constructor</p>
 								<ul>
-									<li>Defined in <a href="https://github.com/mengqiy/kpt-functions-sdk/blob/5bbba38/ts/kpt-functions/src/errors.ts#L20">errors.ts:20</a></li>
+									<li>Defined in <a href="https://github.com/mengqiy/kpt-functions-sdk/blob/2e2d174/ts/kpt-functions/src/errors.ts#L20">errors.ts:20</a></li>
```

Comment:
This should reference your fork.

Reply:
Good catch. It appears `typedoc` picks whatever the `origin` is. I have updated the `CONTRIBUTING.md` doc.

Reply:
I think we should automatically post-process to handle this automatically, otherwise inevitably someone will forget to do this.. But that's outside the scope here.

Reply:
I also noticed the commit SHA used is the parent of the first commit in this PR:

2e2d174

In other words, what HEAD points to when running `doc generate`. We should change CONTRIBUTING doc to say something:

You need to run the generate command on a pristine repo where HEAD points to the current master branch.

Reply:
Created https://github.com/GoogleContainerTools/kpt/issues/2396